pyproject.toml

```diff
@@ -96,6 +97,7 @@ pytest = "7.2.0"
 pytest-cov = "^3.0.0"
 types-PyYAML = "^6.0.12.12"
 types-setuptools = "^69.0.0.0"
+types-aiofiles = "^24.1.0.20240626"
```

Comment:
can we use a less specific version here? (maybe "^24.1.0")

Reply:
going to see if i can update this as a follow-up, still running into `poetry lock --no-update` hanging for me